<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupDetailsWidget</class>
 <widget class="QWidget" name="GroupDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QFrame" name="topBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #34495e;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>15</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2c3e50;
color: white;
border-radius: 5px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>15</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e74c3c;
color: white;
border-radius: 5px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Borrar Grupo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>15</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2ecc71;
color: white;
border-radius: 5px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Crear Grupo</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 32px;
color: black;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Datos del grupo</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
color: #555;</string>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #ccc;
border-radius: 5px;
padding: 5px;</string>
   </property>
   <property name="placeholderText">
    <string>Lorem ipsum</string>
   </property>
  </widget>
  <widget class="QLabel" name="scheduleLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
color: #555;</string>
   </property>
   <property name="text">
    <string>Rango del horario</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fromComboBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #ccc;
border-radius: 5px;
padding: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>Desde</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="toComboBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #ccc;
border-radius: 5px;
padding: 5px;</string>
   </property>
   <item>
    <property name="text">
     <string>Hasta</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="subjectsLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
color: #555;</string>
   </property>
   <property name="text">
    <string>Materias</string>
   </property>
  </widget>
  <widget class="QFrame" name="subjectsFrame">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5c7c9e;
border-radius: 5px;</string>
   </property>
   <widget class="QComboBox" name="quarterComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #ccc;
border-radius: 5px;
padding: 5px;
background-color: white;</string>
    </property>
    <item>
     <property name="text">
      <string>4to Trimestre 2025</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="colorLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;
color: #555;</string>
   </property>
   <property name="text">
    <string>Color identificativo</string>
   </property>
  </widget>
  <widget class="QWidget" name="colorPicker" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 50px;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
  </widget>
  <widget class="QWidget" name="selectedColor" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00ff40;
border-radius: 5px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
